public/testrun.html

```diff
@@ -92,7 +103,7 @@ <h1>
   backArrows.style('display', display)
   if (display == 'none') loadingDiv.html(
     `Click on a directory or file name to trigger a test. ` +
-    `Tests in <span style='color: ${activeColor}'>red</span> are currently imported in 'client/test/internals.js'.`
+    `Tests in <span style='color: ${activeColor}'>red</span> are currently imported in 'client/test/internals.js'.` // don't see any red color. is this outdated?
```

Comment:
check out my question?

Reply:
This red text comment is outdated, from the previous webpack approach. All the spec files are always bundled now every time, but there is an if-else logic to only run specs that matchs the glob pattern in the URL params. 

Previously, selecting a test would trigger a request to the PP server, which triggers a rewrite of `server/test/internals.js`, and then that triggers a webpack rebundling. The current approach is much simpler, no need for a server request to trigger import rewrites and rebundling.